src/taskpane/index.tsx

```diff
@@ -7,21 +7,21 @@ import { FluentProvider, webLightTheme } from "@fluentui/react-components";
 
 const title = "Contoso Task Pane Add-in";
 
-const rootElement: HTMLElement = document.getElementById("container");
-const root = createRoot(rootElement);
+const rootElement: HTMLElement | null = document.getElementById("container");
+const root = rootElement ? createRoot(rootElement) : undefined;
 
 /* Render application after Office initializes */
-Office.onReady(() => {
-  root.render(
+Office.onReady((info) => {
+  root?.render(
     <FluentProvider theme={webLightTheme}>
-      <App title={title} />
+      <App host={info.host} title={title} />
```

Comment:
This `host` prop is never used. Can we remove it?

Reply:
done